src/sentry/utils/query.py

```diff
@@ -69,7 +69,16 @@ class RangeQuerySetWrapper:
     Very efficient, but ORDER BY statements will not work.
     """
 
-    def __init__(self, queryset, step=1000, limit=None, min_id=None, order_by="pk", callbacks=()):
+    def __init__(
+        self,
+        queryset,
+        step=1000,
+        limit=None,
+        min_id=None,
+        order_by="pk",
+        callbacks=(),
+        result_value_getter=None,
```

Comment:
I added this in place so that we could use this wrapper on querysets that aren't actually fetching models